docs/guide/server-rendering.md

```
@@ -0,0 +1,80 @@
+    app-html $ make-string
+      comp-container $ let
+          s schema/store
+        assoc reel-schema/reel :base s :store s
+    styles $ .join-str @*style-list-in-nodejs (str &newline &newline)
+
+  ;nil
+```
+
+### Report bugs
+
+This feature has not been well tested in real world yet. Submit bugs at https://github.com/Respo/respo/issues
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)